lab_1_classify_by_unigrams/start.py

```diff
@@ -2,6 +2,7 @@
 Language detection starter
 """
 # pylint:disable=too-many-locals, unused-argument, unused-variable
+from main import *
```

Comment:
import certain functions

Reply:
no, you can import like `from main import some_function, another_function`

Reply:
done